Cargo.toml

```diff
@@ -21,6 +21,7 @@ license = "MIT OR Apache-2.0"
 clippy = []
 
 [dependencies]
+clap = { version = "4.5.53", features = ["derive"] }
```

Comment:
why this, only `fsays/src/main.rs` depends on `clap`. The `ferris-says` library does not do any commandline parsing